sag_ji4.par

```diff
@@ -1,306 +1,306 @@
+24.662198471110600	)/|	41553/40960		51
+24.885981585695500	,.|)	6656/6561	,.f	51.5
+25.083774513351800	.|)	2097152/2066715	.f	52
+25.645875903006700	`.|)	2079/2048	`.f	53
+26.067451945247900	,,|)	66/65	,,f	54
+26.629553334902900	,|)	65/64	,f	55
+27.051129377144100	|)	64/63	f	56
+27.613230766799000	`|)	20000/19683	`f	57
+28.034806809040200	``|)	216513/212992	``f	58
+28.596908198695100	,'|)	2132325/2097152	,'f	59
+29.018484240936400	'|)	3645/3584	'f	60
+29.580585630591300	`'|)	3125/3072	`'f	61
+30.002161672832500	,)|)	2720/2673		62
+30.564263062487400	)|)	57/56		63
+30.985839104728600	.(|	56/55	.?	64
+31.407415146969900	|\	55/54	y	65
+31.969516536624800	`|\	22528/22113	`y	66
+32.391092578866000	,(|	375/368	,?	67
+32.953193968520900	(|	45927/45056	?	68
+33.374770010762100	`(|	52/51	`?	69
+33.936871400417100	``(|	51/50	``?	70
+34.358447442658300	,'(|	252/247	,'?	71
+34.920548832313200	'(|	50/49	'?	72
+35.118091464366500	,~|)	248/243		72.5
+35.342124874554400	~|)	49/48		73
+35.904226264209400	`~|)	334611/327680		74
+36.325802306450600	``~|)	48/47		75
+36.588490003137900	,.(|(	24057/23552	,.g	75.5
+36.887903696105500	.(|(	8192/8019	.g	76
+37.309479738346700	'~|)	535815/524288		77
+37.871581128001700	/|~	46/45		78
+38.061940349778500	,,(|(	1701/1664	,,g	78.5
+38.293157170242900	,(|(	1408/1377	,g	79
+38.855258559897800	(|(	45/44	g	80
+39.276834602139000	`'(|(	1024/1001	`g	81
+39.698410644380200	~|\	16767/16384		82
+40.260512034035200	,.//|	819/800	,.=	83
+40.682088076276400	.//|	128/125	.=	84
+41.244189465931300	`.//|	2240/2187	`.=	85
+41.665765508172500	,,//|	459/448	,,=	86
+42.227866897827500	,//|	5373459/5242880	,=	87
+42.649442940068700	//|	6561/6400	=	88
+43.211544329723600	`//|	525/512	`=	89
+43.633120371964800	``//|	40/39	``=	90
+44.195221761619800	,'//|	35721/34816	,'=	91
+44.616797803861000	'//|	39/38	'=	92
+45.178899193515900	`'//|	77/75	`'=	93
+45.600475235757100	,)//|	54675/53248		94
+46.162576625412100	)//|	416/405		95
+46.584152667653300	`)//|	131072/127575		96
+47.146254057308200	``)//|	37/36		97
+47.567830099549400	,,/|)	18711/18200	,,%	98
+48.129931489204400	,/|)	1053/1024	,%	99
+48.551507531445600	/|)	36/35	%	100
+48.973083573686800	`/|)	250/243	`%	101
+49.535184963341700	(|~	176/171		102
+49.956761005582900	,'/|)	34543665/33554432	,'%	103
+50.518862395237900	'/|)	59049/57344	'%	104
+50.940438437479100	`'/|)	16875/16384	`'%	105
+51.219540248855700	./|\	11264/10935	.^	105.5
+51.502539827134000	`./|\	34/33	`.^	106
+51.924115869375200	,,/|\	640/621	,,^	107
+52.486217259030200	,/|\	567/550	,^	108
+52.907793301271400	/|\	33/32	^	109
+53.469894690926300	`/|\	185895/180224	`^	110
+53.891470733167500	,(/|	65/63		111
+54.453572122822500	(/|	4096/3969		112
+54.875148165063700	`(/|	32/31		113
+55.095545776914000	'/|\	1082565/1048576	'^	113.4
+55.437249554718600	`'/|\	24057/23296	`'^	114
+55.858825596959800	,)/|\	595/576		115
+56.420926986614700	)/|\	405/392		116
+56.842503028856000	`)/|\	1323/1280		117
+57.264079071097200	``)/|\	19683/19040		118

```

Comment:
interestingly, this was fine, though there was a mistake in the downwards section